CMakeLists.txt

```diff
@@ -30,6 +30,16 @@ else()
     add_subdirectory(${PICO_SDK_PATH}/src/host/pico_platform pico_platform)
 
     add_executable(picotool main.cpp)
+    set(PICOTOOL_VERSION 1.1.0-develop)
+    set(SYSTEM_VERSION "${CMAKE_SYSTEM_NAME} ${CMAKE_SYSTEM_VERSION}")
+    set(COMPILER_INFO "${CMAKE_C_COMPILER_ID}-${CMAKE_C_COMPILER_VERSION} ${CMAKE_BUILD_TYPE}")
```

Comment:
When I run `./picotool version` (on Linux) I get `picotool v1.1.0-develop (Linux 5.4.0-77-generic, GNU-7.5.0 )` - does this mean that the `${CMAKE_BUILD_TYPE}` variable needs to have a default value given to it, if not explicitly set?

Reply:
ah, i guess i always build from my idea which sets it explicitly

Reply:
fixed